src/models/types.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
Can you move this file to `./constants/model-types`? And then also change the existing `./constants/types` to be called `./constants/transfer-types`? Thanks!